cpp/src/barretenberg/honk/transcript/transcript.test.cpp

```diff
@@ -87,7 +89,7 @@ template <typename Flavor> class TranscriptTests : public testing::Test {
         round++;
         // TODO(Mara): Make testing more flavor agnostic so we can test this with all flavors
         if constexpr (IsGrumpkinFlavor<Flavor>) {
-            manifest_expected.add_entry(round, "IPA:poly_degree", circuit_size);
+            manifest_expected.add_entry(round, "IPA:poly_degree", size_uint64);
```

Comment:
This was a bug in the test that was revealed by the work in this PR but is otherwise unrelated.